upload-cloud-storage/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Cloud Storage Uploader
+description: Upload files or folders to GCS buckets
+author: GoogleCloudPlatform
+inputs:
+  service_account_key:
```

Comment:
The secret manager action uses `credentials` to be more generic. What do you think about using that here too?

Reply:
switched to credentials